### PR TITLE
no-std compatibility

### PR DESCRIPTION
**Pull request by vlopes11**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -72,7 +69,9 @@ mod fp12;
 mod fp6;
 
 // The BLS parameter x for BLS12-381 is -0xd201000000010000
+#[cfg(feature = "std")]
 const BLS_X: u64 = 0xd201000000010000;
+#[cfg(feature = "std")]
```
Comment (CPerezz) [inline src/lib.rs:74]: Why these constants are only `std` avaliable?

**Review by CPerezz — CHANGES_REQUESTED** [review]
Appart from the question, I have a request. 
Could you please add tests for `default features` and another test for `no-default-features` in the CI?

On that way, we know that everything works. Since MultiscalarMul and other stuff are only `std` avaliable. But we want to be sure that the no_std still works as expected.

File: src/lib.rs (hunk 1 above)
Comment (vlopes11) [reply src/lib.rs]: Fixed. was part of a local test and this feature constaint is indeed not required

Comment (CPerezz) [reply src/lib.rs]: KK perfect!

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.0] - 08-11-20
```
Comment (CPerezz) [inline CHANGELOG.md:10]: Shouldn't it be 0.2.0?

Comment (CPerezz) [reply CHANGELOG.md:10]: CHANGELOG not updated. Nevermind

**Review by CPerezz — APPROVED** [review]
LGTM

**Pull request merged**